src/shared/dropdown-list/dropdown-search.styles.tsx

```diff
@@ -12,29 +24,43 @@ export const Container = styled.li`
     align-items: center;
 `;
 
-export const SearchInput = styled(BasicInput)`
-    height: 3rem;
+export const SearchInput = styled(BasicInput)<InputStyleProps>`
+    // 2.5rem 40px, 3rem 48px
+    height: ${(props) => (props.$variant === "small" ? 2.5 : 3)}rem;
     flex: 1;
     padding: 0 0.5rem 0 0;
     width: 100%;
 `;
 
-export const SearchIcon = styled(MagnifierIcon)`
-    height: 1.375rem;
-    width: 1.375rem;
+export const SearchIcon = styled(MagnifierIcon)<IconProps>`
+    ${(props) => {
+        return getCssHeightAndWidth(props.$variant);
+    }}
     margin: 0 0.5rem;
     color: ${Color.Neutral[3]};
 `;
 
-export const CancelButton = styled(ClickableIcon)`
+export const CancelButton = styled(ClickableIcon)<IconProps>`
+    ${(props) => {
+        return getCssHeightAndWidth(props.$variant);
+    }}
     padding: 0;
     margin: 0 0.5rem;
     color: ${Color.Neutral[3]};
     cursor: pointer;
+    ${(props) => {
+        if (props.$variant === "small")
+            return css`
+                svg {
+                    ${getCssHeightAndWidth(props.$variant)}
+                }
+            `;
+    }}
```

Comment:
actually this is redundant as the CancelIcon has its own css styling already

Reply:
i added this because its ClickableIcon set default svg height and width to 1.125rem
https://github.com/LifeSG/react-design-system/blob/55f2a59308303af3298c7c44a1ad0ba05e713eed/src/shared/clickable-icon/clickable-icon.style.tsx#L23

hmm shall i modify ClickableIcon to have the variant prop too?

Reply:
then in that case, CancelButton can handle the `svg` styling for both variants, and CancelIcon can be removed entirely (the component can use CrossIcon directly instead)

I'd say ClickableIcon doesn't need a variant for now because there's actually no standardised sizing for icons